webapp/package.json

```diff
@@ -9,7 +9,8 @@
         "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix",
         "test": "jest --forceExit --detectOpenHandles --verbose",
         "test:watch": "jest --watch",
-        "test-ci": "jest --forceExit --detectOpenHandles --maxWorkers=2"
+        "test-ci": "jest --forceExit --detectOpenHandles --maxWorkers=2",
```

Comment:
Note that we'll also want to sync with the starter template the scripts in `package.json` as well. I can close https://github.com/mattermost/mattermost-plugin-gitlab/pull/421 in favor of this PR